WPT Import job has been failing since last month

Most recent failure:

https://github.com/servo/servo/actions/runs/6286825646/job/17071266200

```
Error running mach:

    ['update-wpt', '--legacy-layout', 'wpt-logs-linux-layout-2013/test-wpt.1.log', 'wpt-logs-linux-layout-2013/test-wpt.10.log', 'wpt-logs-linux-layout-2013/test-wpt.11.log', 'wpt-logs-linux-layout-2013/test-wpt.12.log', 'wpt-logs-linux-layout-2013/test-wpt.13.log', 'wpt-logs-linux-layout-2013/test-wpt.14.log', 'wpt-logs-linux-layout-2013/test-wpt.15.log', 'wpt-logs-linux-layout-2013/test-wpt.16.log', 'wpt-logs-linux-layout-2013/test-wpt.17.log', 'wpt-logs-linux-layout-2013/test-wpt.18.log', 'wpt-logs-linux-layout-2013/test-wpt.19.log', 'wpt-logs-linux-layout-2013/test-wpt.2.log', 'wpt-logs-linux-layout-2013/test-wpt.20.log', 'wpt-logs-linux-layout-2013/test-wpt.3.log', 'wpt-logs-linux-layout-2013/test-wpt.4.log', 'wpt-logs-linux-layout-2013/test-wpt.5.log', 'wpt-logs-linux-layout-2013/test-wpt.6.log', 'wpt-logs-linux-layout-2013/test-wpt.7.log', 'wpt-logs-linux-layout-2013/test-wpt.8.log', 'wpt-logs-linux-layout-2013/test-wpt.9.log']

The error occurred in code that was called by the mach command. This is either
a bug in the called code itself or in the way that mach is calling it.
You can invoke |./mach busted| to check if this issue is already on file. If it
isn't, please use |./mach busted file| to report it. If |./mach busted| is
misbehaving, you can also inspect the dependencies of bug 1543241.

If filing a bug, please include the full output of mach, including this error
message.

The details of the failure are as follows:

ModuleNotFoundError: No module named 'mozleak'

  File "/home/runner/work/servo/servo/python/servo/testing_commands.py", line 406, in update_wpt
    return wpt.update.update_tests(**kwargs)
  File "/home/runner/work/servo/servo/python/wpt/update.py", line 110, in update_tests
    wptcommandline.set_from_config(kwargs)
  File "/home/runner/work/servo/servo/tests/wpt/tests/tools/wptrunner/wptrunner/wptcommandline.py", line 468, in set_from_config
    kwargs["product"] = products.Product(kwargs["config"], kwargs["product"])
  File "/home/runner/work/servo/servo/tests/wpt/tests/tools/wptrunner/wptrunner/products.py", line 20, in __init__
    module = product_module(config, product)
  File "/home/runner/work/servo/servo/tests/wpt/tests/tools/wptrunner/wptrunner/products.py", line 11, in product_module
    module = importlib.import_module("wptrunner.browsers." + product)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/runner/work/servo/servo/tests/wpt/tests/tools/wptrunner/wptrunner/browsers/firefox.py", line 15, in <module>
    import mozleak
Error: Process completed with exit code 1
```

--- a/python/wpt/manifestupdate.py
+++ b/python/wpt/manifestupdate.py
@@ -34,6 +34,7 @@ def create_parser():
 def update(check_clean=True, rebuild=False, **kwargs):
     logger = wptlogging.setup(kwargs, {"mach": sys.stdout})
     kwargs = {"config": os.path.join(WPT_PATH, "config.ini"),
+              "product": "servo",
               "manifest_path": os.path.join(WPT_PATH, "meta"),
               "tests_root": None,
               "metadata_root": None}

--- a/python/wpt/update.py
+++ b/python/wpt/update.py
@@ -103,8 +103,8 @@ def update_tests(**kwargs) -> int:
         if key not in args or args[key] is None:
             args[key] = value
 
-    set_if_none(kwargs, "product", "servo")
     set_if_none(kwargs, "config", os.path.join(WPT_PATH, "config.ini"))
+    kwargs["product"] = "servo"
     kwargs["store_state"] = False
 
     wptcommandline.set_from_config(kwargs)

